lib_xud/src/core/included/XUD_Token_In_DI.S

```diff
@@ -20,84 +19,80 @@
+    bf         r4, XUD_IN_NotReady
+    
+    ldaw       r1, dp[g_stallTable_IN]
+    ldw        r11, r1[r10]
+    bt         r11, XUD_IN_NotReady                // Note, user could potentially stall an ISO EP.
+
     ldw        r11, r4[4]                          // Load PID from structure
 
-InReady:
+XUD_IN_Ready:
     ldw        r8, r4[3]                           // Load buffer
     ldw        r6, r4[7]                           // Load tail length (bytes)
-    ldw        r4, r4[6]                           // Load data length
-    bf         r4, SmallTxPacket                   // Check for "short" packet
+    ldw        r4, r4[6]                           // Load data length (words)
+    bf         r4, XUD_IN_SmallTxPacket            // Check for Short packet
 
 XUD_IN_Tx:
     outpw      res[TXD], r11, 8                    // Out PID
     ldw        r11, r8[r4]                         // Load first data word
 
 XUD_IN_TxPid:
-    out        res[TXD], r11
-    crc32_inc  r7, r11, r9, r4, 1
-    bf         r4, XUD_IN_TxLoopEnd
+    out         res[TXD], r11
+    crc32_inc   r7, r11, r9, r4, 1
+    bf          r4, XUD_IN_TxLoopEnd
 
 XUD_IN_TxLoop:
-    ldw        r11, r8[r4]                         // Load first data word
-    crc32_inc  r7, r11, r9, r4, 1
-    {out       res[TXD], r11;     bt       r4, XUD_IN_TxLoop}
+    ldw         r11, r8[r4]                         // Load first data word
+    crc32_inc   r7, r11, r9, r4, 1
+    {out         res[TXD], r11;     bt       r4, XUD_IN_TxLoop}
 
 XUD_IN_TxLoopEnd:
-    ldw        r11, r8[r4]                         // Load tail
-    crcn       r7, r11, r9, r6
-    outpw      res[TXD], r11, r6
+    ldw         r11, r8[r4]                         // Load tail
+    crcn        r7, r11, r9, r6
 
 XUD_IN_TxLoopEnd_CrcCalc:
-    crc32      r7, r4, r9                          // r4: 0 (from bf)
-    not        r7, r7
+    crc32       r7, r4, r9                          // r4: 0 (from bf)
+    outpw       res[TXD], r11, r6
+    not         r7, r7
 XUD_IN_TxCrc:
-    outpw      res[TXD], r7, 16                    // Output 16-bit CRC
+    outpw       res[TXD], r7, 16                    // Output 16-bit CRC
 
 .scheduling default
 
 // Wait for handshake... or timeout
 DoneTail:
     ldw         r11, sp[STACK_EPTYPES_IN]
-    ldw         r11, r11[r10]                      // Load EP Type
+    ldw         r11, r11[r10]                       // Load EP Type
     bt          r11, SetupReceiveHandShake
-    bu          DoneTx
+    bu          XUD_IN_DoneTx
 
 SetupReceiveHandShake:
     ldc        r11, 8
     setpsc     res[RXD], r11                       // Set port shift count (expect 8 bit handshake)
-    eeu        res[RXD]                            // Events on RXD always enabled - Can;t be any more due to using events on channels
+    eeu		   res[RXD]                            // Events on RXD always enabled - Can;t be any more due to using events on channels
```

Comment:
Quite a lot of white space here.  Is that intentional?

Reply:
Will fix on my next PR